examples/gnn_explainer.py

```diff
@@ -57,3 +58,7 @@ def forward(self, x, edge_index, edge_weight):
 explanation = explainer(data.x, data.edge_index, index=node_index,
                         edge_weight=data.edge_weight)
 print(f'Generated explanations in {explanation.available_explanations}')
+
+ax = explainer.visualize_feature_importance(explanation)
+print('Plotting the feature importance')
+plt.savefig("feature_importance.png")
```

Comment:
I'd like to see how this looks :). Could you paste it here?

Reply:
Here's an example: 
![feature_importance](https://user-images.githubusercontent.com/36218572/206197340-d01edb0d-9769-4956-bdcd-bb75708966f8.png)

Reply:
FYI running this with a GNNExplainer of 10 epochs to expedite the process so importances are very similar

Reply:
Great thanks for sharing.